controller/bindings/screen-settings/screen_settings.ml

```diff
@@ -0,0 +1,61 @@
+open Lwt
+
+let log_src = Logs.Src.create "screen-scaling"
+let settings_file = "/var/lib/gui-localization/screen-scaling"
+
+(* Scaling options *)
+type scaling =
+  (* No config file
+     xsession script decides behaviour
+  *)
+  | Default
+  (* Config file with keyword: "scaled"
+     Explicitly opt-in to scaling to FullHD
+  *)
+  | Scaled
```

Comment:
I chose the ambiguous name, because in case we were scaling a different aspect ratio, `FullHD` would be slightly misleading. I agree that if we'd have other specific scalings, the ambiguity would become problematic, so I agree with your suggestion.

For our purposes we can think of "Full HD", as anything with 1080px vertically even if it's a different aspect ratio.